src/components/Search/CompareOverTime.tsx

```diff
@@ -182,7 +186,14 @@ function CompareOverTime({
             className={`${dropDownStyles.dropDown} ${bottomStyles.container}`}
           >
             <div className='bottom-dropdowns'>
-              <FrameworkDropdown compact={true} />
+              <FrameworkDropdown
+                compact={true}
+                frameworkId={frameworkId}
+                onChange={(event: SelectChangeEvent) => {
+                  const id = +event.target.value as Framework['id'];
```

Comment:
as a quick passing-by comment, this work could have been done inside `FrameworkDropdown`. Several reasons:
* every user of FrameworkDropdown has to do it
* this is really the secret sauce of the FrameworkDropdown: users do not need to know FrameworkDropdown is implemented as a select entry, getting the id as an DOM event handler.
* the input prop `frameworkId` is a `Framework['id']` so the output prop (that we get in `onChange`) should be the same type for consistency

but that's not super important and can be changed also later.